api/controllers/generateBadges.py

```diff
@@ -124,3 +124,13 @@ def delete_badge(badgeId):
     deleteFile('badges/' + badgeId + '.pdf')
     badge.delete_from_db()
     return jsonify(DeletedBadges().dump(temp_badge).data)
+
+
+@router.route('/change/name', methods=['PATCH'])
```

Comment:
Can you change the name of the route to comply with the model route for the generated badge.